packages/frosted-ui-icons/src/AlignJustify12.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { IconProps } from './types';
+
+export const AlignJustify12 = React.forwardRef<SVGSVGElement, IconProps>(
+  ({ color = 'currentColor', ...props }, forwardedRef) => {
+    return (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlSpace="preserve"
+        width="12"
+        height="12"
+        {...props}
+        ref={forwardedRef}
+      >
+        <path
+          d="M2 2.1h8M2 4.7h8M2 10h8M2 7.4h8"
+          stroke={color}
+          strokeWidth="1.5"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+      </svg>
+    );
+  },
+);
+
+export default AlignJustify12;
```

Comment:
We shouldn't add icons manually. The icons should be added to our [Figma file](https://www.figma.com/design/Qz3t7Yi565N2u4l3Ha1w99/Frosted-Product-Icons-%F0%9F%9F%A2?node-id=204-125&node-type=frame&m=dev) and then pulled in by our script